examples/fscloud/variables.tf

```diff
@@ -50,3 +50,13 @@ variable "access_tags" {
   description = "Optional list of access tags to be added to the created resources"
   default     = []
 }
+
+variable "management_endpoint_type_for_bucket" {
+  description = "The type of endpoint for the IBM terraform provider to use to manage the bucket. (public, private or direct)"
+  type        = string
+  default     = "private"
```

Comment:
Wondering if this should default to `public` so would work locally by default? And in [catalogValidationValues.json.template ](https://github.com/terraform-ibm-modules/terraform-ibm-cos/blob/main/examples/fscloud/catalogValidationValues.json.template) we can set it to private as we know that will always run in schematics?

Reply:
I figured we should default to secure in fscloud and override for our tests. I was thinking, if the CBR is already only from inside VPC, then how does access from github actions work over public? So I am doing some extra checking.